kivymd/uix/textfield.py

```diff
@@ -1271,6 +1271,7 @@ def __init__(self, **kwargs):
         self._lbl_icon_right = MDIcon(theme_text_color="Custom")
         super().__init__(**kwargs)
         self.cursor_color = self.theme_cls.primary_color
+        self.icon_left_color = self.theme_cls.text_color
```

Comment:
This only works if we set and remove focus from the text field. By default, the icon color is still not set correctly:

![5](https://user-images.githubusercontent.com/16930280/100012206-f4092c00-2de3-11eb-892e-1f977e11b900.gif)

Reply:
I'm sure there's some patch mismatch while testing on your end. It's working fine.
![ezgif com-video-to-gif(1)](https://user-images.githubusercontent.com/37111736/100039494-5afe0380-2e2b-11eb-9d6d-a12642a0876f.gif)

Reply:
Yes, sorry.

Reply:
No worries ;)